README.md

```diff
@@ -63,13 +63,21 @@ mode and editor commands, making the best use of both editors.
 -   If you want to use Neovim from WSL, set the `useWSL` configuration toggle and specify Linux path to nvim binary.
     `wsl.exe` Windows binary and `wslpath` Linux binary are required for this. `wslpath` must be available through
     `$PATH` Linux env setting. Use `wsl --list` to check for the correct default Linux distribution.
--   Add to your `settings.json`:
+-   Assign [affinity](#affinity) value for performance improvement.
 
-```json
-"extensions.experimental.affinity": {
-    "asvetliakov.vscode-neovim": 1
-},
-```
+    -   Settings > Features > Extensions > Experimental Affinity.
+
+            Assign value 1 for entry `asvetliakov.vscode-neovim`
+            ![Alt text](images/experimental_affinity.png)
```

Comment:
Don't think an image is worth adding to the repo. Eventually vscode will get a proper doc for this, then we'll link to it. This is just temporary.

Reply:
Makes sense. Removed the image.